### ISSUE TITLE
[2.0.4-r2 (106)] Samsung Galaxy S6 / 6.0.1 - Lomotif is stop playing under "Following" tab when user tap multiple times on "Like" option.

### ISSUE THREAD

**Issue by Aboli-Testershub** [Lomotif/lomotif-android-issues#974]
Frequency: Everytime

Action:
1: Launch the app and login with valid credentials.
2: Tap on "Feed>>Following" tab.
3: Tap multiple times on "Like" option of any lomotif.
4: Check if lomotif is stop playing.

Expected result: Lomotif  should not stop playing under "Following" tab when user tap multiple times on "Like" option.

Actual result: 
1: Lomotif  is stop playing under "Following" tab when user tap multiple times on "Like" option.
2: Sometimes lomotif is displayed as blur and black when user tap multiple times on "Like" option under "Following" tab.

**Note: Similar issue is observed on [Live Build_2.0.4]**

**Comment by Pankaj-Testershub** [comment]
@xiuie I can reproduce this issue for build [2.0.4-r2 (106)] on Samsung Galaxy Note 4 / 6.0.1

**Comment by xiuie** [comment]
After it stops, does it start again? Or hangs forever? @Aboli-Testershub @Pankaj-Testershub This will help me judge severity of the bug. For now I assume if you tap again, it may start playing eventually.

**Comment by Pankaj-Testershub** [comment]
@xiuie Yes, it starts playing again when user refresh the "Following" tab or navigate back to "Following" tab from any other screen.
